frontend/src/constants/Routes.ts

```diff
@@ -55,6 +50,12 @@ export const ASSESSMENT_EDITOR_QUESTION_PREVIEW_PAGE = ({
   ASSESSMENT_EDITOR_QUESTION_EDITOR_BASE({ assessmentId, questionIndex }) +
   "/preview";
 
+export const ASSESSMENT_PREVIEW_PAGE = ({
+  assessmentId,
+}: {
+  assessmentId?: string;
+}) => "/admin/assessment/" + assessmentId + "/preview";
```

Comment:
I think it would be kind of nice to overload the route here... i.e `/admin/assessments/ + assessmentId + /preview`, but I can't figure out why that isn't working... let me know if you have thoughts (:

Reply:
Yeah routing is tricky for this page... Happy to pair at some point if that would help. Or I can look into what's happening later this week

Reply:
Yah, we can pair on Sunday? This PR isn't a blocker so I don't mind leaving it open

Reply:
Same here, we resolved this synchronously right? 😅